CHARTER.md

```diff
@@ -1,107 +1,121 @@
 
-   - v. appointing representatives to work with other open source or open standards communities;
+- d. A Technical Initiative may be de-scoped or archived for any reason following confirmation on the proposal with a 2/3 majority approval from the Technical Steering Committee. That proposal must include an update to this document.
 
-   - vi. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
+- e. The current Technical Initiatives in scope for this WG are:
 
-   - vii. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the Technical Initiative; and
+  - Open Source Project Security Baseline
+  - Open Source Project Security Assessments
+  - Security Insights Specification
 
-   - viii. coordinating any communications regarding the Technical Initiative.
+## 4. Voting
 
-#### 3. TSC Voting
+- a. While the WG aims to operate as a consensus-based community, if any TSC decision requires a vote to move the WG forward, the voting members of the TSC will vote on a one vote per voting member basis.
 
-- a. While the Technical Initiative aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Technical Initiative forward, the voting members of the TSC will vote on a one vote per voting member basis.
+- b. Quorum for a live vote requires at least fifty percent of all voting members of the TSC to be present.
 
-- b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+- c. Any TSC member may determine that a decision requires a community consensus or TSC vote, while others may always require a vote by nature of this charter or by custom of the WG.
 
-- c. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
+- d. Bypassing a vote required by charter or well-established custom is grounds for removal from the TSC, subject to either a TSC vote or community consensus.
 
-- d. In the event a vote cannot be resolved by the TSC, any voting member of the TSC may refer the matter to the TAC for assistance in reaching a resolution.
+- e. Except as provided elsewhere in this charter, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
```

Comment:
Is there a need at all for "live voting". Could this just be simplified to electronic votes? I think that would be more inclusive for folks unable to attend a specific meeting for voting purposes.

Reply:
I'm fine with removing it.